example/build.gradle

```diff
@@ -50,7 +50,7 @@ private def getValue(key) {
 }
 
 dependencies {
-    implementation project(':stripe')
+    implementation project(':payment-core')
```

Comment:
Can we use nested modules, like `:stripe:payment:core`?

Reply:
it is possible to nest them, but it seems nested modules are suitable when they are siblings features like [this](http://www.developerphil.com/nested-modules-in-gradle/), however for `payment` and `payment-core` they're not siblings, but one depending on another, we'll have a dir structure like this:
```
stripe-android
  payment // payment module
    src // payment src
    res // payment res
    build.gradle // payment build.gradle, depends on :core
    core // core module, nested inside payment
      src  // core src
      res // core res
      build.gradle // core build.gradle
```

I think nesting them would make the structure less readable